--- texlive.tlpdb.orig	2012-12-06 07:38:57.246579879 +0100
+++ texlive.tlpdb	2012-12-06 07:38:57.275579821 +0100
@@ -76,7 +76,7 @@
 catalogue-ctan /macros/latex/contrib/abc
 catalogue-date 2008-03-08 20:47:21 +0100
 catalogue-license lppl
-catalogue-version .0
+catalogue-version 0
 
 name abraces
 category Package
@@ -6963,7 +6963,7 @@
 catalogue-license pd
 catalogue-version 1.0a
 
-name Asana-Math
+name asana-math
 category Package
 revision 27228
 shortdesc A font to typeset maths in Xe(La)TeX and Lua(La)TeX.
@@ -24675,7 +24675,6 @@
 shortdesc TeX Live documentation
 relocated 1
 depend texlive-common
-depend texlive-docindex
 depend texlive-en
 
 name collection-documentation-bulgarian
@@ -24974,7 +24973,7 @@
 revision 28412
 shortdesc Extra fonts
 relocated 1
-depend Asana-Math
+depend asana-math
 depend adforn
 depend adfsymbols
 depend allrunes
@@ -25092,7 +25091,7 @@
 depend libertine
 depend librebaskerville
 depend libris
-depend linearA
+depend lineara
 depend lxfonts
 depend ly1
 depend mathabx
@@ -25451,7 +25450,7 @@
 depend cns
 depend ctex
 depend dnp
-depend fonts-tlwg
+depend tlwg
 depend garuda-c90
 depend hyphen-chinese
 depend ipaex
@@ -26969,7 +26968,7 @@
 depend m-tx
 depend musixguit
 depend musixtex
-depend musixtex-fonts
+depend musixtex-fnts
 depend pmx
 depend songbook
 depend collection-latex
@@ -27408,25 +27407,6 @@
 depend texinfo
 depend collection-basic
 
-name collection-texworks
-category Collection
-revision 25704
-shortdesc TeXworks editor
-relocated 1
-longdesc See http://tug.org/texworks.
-depend collection-basic
-depend texworks
-
-name collection-wintools
-category Collection
-revision 13822
-shortdesc Windows support programs
-relocated 1
-longdesc Utilities for Windows, since they are not readily available
-longdesc there: chktex, unzip, wget, xpdf, and the dviout previewer.
-depend wintools.win32
-depend dviout.win32
-
 name collection-xetex
 category Collection
 revision 28124
@@ -48724,7 +48704,7 @@
 catalogue-license lppl1.3
 catalogue-version v2.2b
 
-name fonts-tlwg
+name tlwg
 category Package
 revision 25417
 shortdesc Thai fonts for LaTeX from TLWG.
@@ -59704,13 +59684,6 @@
 catalogue-license lppl
 catalogue-version v1.6b
 
-name installfont.i386-linux
-category Package
-revision 19743
-shortdesc i386-linux files of installfont
-binfiles arch=i386-linux size=1
- bin/i386-linux/installfont-tl
-
 name interactiveworkbook
 category Package
 revision 15878
@@ -68956,7 +68929,7 @@
 catalogue-ctan /macros/latex/contrib/labels
 catalogue-date 2009-09-29 00:01:21 +0200
 catalogue-license lppl1.2
-catalogue-version .13
+catalogue-version 13
 
 name lacheck
 category TLCore
@@ -74805,7 +74778,7 @@
 catalogue-date 2007-01-08 23:14:30 +0100
 catalogue-license lppl
 
-name linearA
+name lineara
 category Package
 revision 15878
 shortdesc Linear A script fonts.
@@ -77037,7 +77010,7 @@
 catalogue-ctan /info/lshort/persian
 catalogue-date 2009-11-09 23:05:00 +0100
 catalogue-license pd
-catalogue-version 4.26:2009-08-04
+catalogue-version 4.26-2009-08-04
 
 name lshort-polish
 category Package
@@ -85890,7 +85863,7 @@
 catalogue-license lppl1.3
 catalogue-version 1.2.2
 
-name musixtex-fonts
+name musixtex-fnts
 category Package
 revision 24517
 shortdesc Fonts used by MusixTeX.
@@ -101006,7 +100979,7 @@
 catalogue-ctan /graphics/pstricks/contrib/pst-layout
 catalogue-date 2011-05-27 12:35:46 +0200
 catalogue-license lppl
-catalogue-version .95
+catalogue-version 95
 
 name pst-lens
 category Package
@@ -120476,7 +120449,6 @@
 revision 13663
 shortdesc i386-linux files of texlive-scripts
 binfiles arch=i386-linux size=2
- bin/i386-linux/man
  bin/i386-linux/rungs
 
 name texlive-sr
@@ -129581,7 +129553,7 @@
 catalogue-ctan /macros/latex209/contrib/trees/tree-dvips
 catalogue-date 2011-03-14 14:09:23 +0100
 catalogue-license lppl1
-catalogue-version .91
+catalogue-version 91
 
 name treetex
 category Package
@@ -140894,7 +140866,6 @@
 revision 26912
 shortdesc i386-linux files of xetex
 binfiles arch=i386-linux size=4837
- bin/i386-linux/teckit_compile
  bin/i386-linux/xdvipdfmx
  bin/i386-linux/xelatex
  bin/i386-linux/xetex
